backend/papersfeed/tests/tests_replies.py

```diff
@@ -0,0 +1,236 @@
+        client.post('/api/reply/collection',
+                    json.dumps({
+                        constants.ID: collection_id,
+                        constants.TEXT: 'test_reply_1'
+                    }),
+                    content_type='application/json')
+
+        client.post('/api/reply/review',
+                    json.dumps({
+                        constants.ID: review_id,
+                        constants.TEXT: 'test_reply_1'
+                    }),
+                    content_type='application/json')
+
+    def test_make_reply(self):
+        """MAKE Reply"""
+        client = Client()
+
+        # Sign In
+        client.get('/api/session',
+                   data={
+                       constants.EMAIL: 'swpp@snu.ac.kr',
+                       constants.PASSWORD: 'iluvswpp1234'
+                   },
+                   content_type='application/json')
+
+        collection_id = Collection.objects.filter(title='test_collection_1').first().id
+        review_id = Review.objects.filter(title="test_review_1").first().id
+
+        # Make Reply Collection
+        response = client.post('/api/reply/collection',
+                               json.dumps({
+                                   constants.ID: collection_id,
+                                   constants.TEXT: 'test_reply_2'
+                               }),
+                               content_type='application/json')
+        self.assertEqual(response.status_code, 201)
+
+        # Make Reply Review
+        response = client.post('/api/reply/review',
+                               json.dumps({
+                                   constants.ID: review_id,
+                                   constants.TEXT: 'test_reply_2'
+                               }),
+                               content_type='application/json')
+        self.assertEqual(response.status_code, 201)
+
+    def test_get_replies(self):
+        """GET REPLY"""
+        client = Client()
+
+        # Sign In
+        client.get('/api/session',
+                   data={
+                       constants.EMAIL: 'swpp@snu.ac.kr',
+                       constants.PASSWORD: 'iluvswpp1234'
+                   },
+                   content_type='application/json')
+
+        user_id = User.objects.filter(email='swpp@snu.ac.kr').first().id
+        collection_id = Collection.objects.filter(title='test_collection_1').first().id
+        review_id = Review.objects.filter(title="test_review_1").first().id
+        collection_reply_id = ReplyCollection.objects.filter(collection_id=collection_id).first().reply_id
+
+        # Get Replies Collection
+        response = client.get('/api/reply/collection',
+                              data={
+                                  constants.ID: collection_id
+                              },
+                              content_type='application/json')
+
+        self.assertEqual(response.status_code, 200)
+        self.assertIn(
```

Comment:
I think if you had used assertJSONEqual, it would have been easier.(I don’t suggest you change this for now.) Please refer to my other tests.

Reply:
Oh I didn't know that! Thank you!